contracts/utils/ReentrancyGuard.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (security/ReentrancyGuard.sol)
+// Modified to use StorageSlot
+pragma solidity ^0.8.12;
+
+import "@openzeppelin/contracts/utils/StorageSlot.sol";
```

Comment:
@drortirosh What is the point of copying the source code of 'ReentrancyGuard.sol' into the 'account-abstraction' repo if it still imports a file ('StorageSlot.sol') presumably from 'node_modules?

Reply:
1. There are no "openzeppelin" imports from the core, only from the samples
2. there implementation doesn't use their own "StorageSlots", but use global members